pkg/model/piped.proto

```diff
@@ -62,7 +62,10 @@ message Piped {
     // The latest connection status of piped.
     ConnectionStatus status = 11 [(validate.rules).enum.defined_only = true];
     // The public key/service account for encrypting the secret data.
-    SealedSecretEncryption sealed_secret_encryption = 12;
+    // Deprecated.
+    // TODO: Remove sealed_secret_encryption field in the future.
+    SecretEncryption sealed_secret_encryption = 12;
```

Comment:
You can instead use the deprecated option to indicate deprecated: https://developers.google.com/protocol-buffers/docs/proto3#options
```suggestion
    SecretEncryption sealed_secret_encryption = 12 [deprecated = true];
```

Reply:
Sure. Applied.